internal/server/data/data.go

```diff
@@ -227,9 +227,10 @@ func initialize(db *DB) error {
 	switch {
 	case errors.Is(err, internal.ErrNotFound):
 		org = &models.Organization{
-			Model:     models.Model{ID: defaultOrganizationID},
-			Name:      "Default",
-			CreatedBy: models.CreatedBySystem,
+			Model:               models.Model{ID: defaultOrganizationID},
+			Name:                "Default",
+			CreatedBy:           models.CreatedBySystem,
+			AllowedLoginDomains: []string{},
```

Comment:
The default organization created through config will never need `AllowedLoginDomains` since social login is only available in the hosted Infra offering.